bedrock/security/templates/security/bug-bounty/faq-webapp.html

```diff
@@ -97,9 +97,35 @@ <h2>Eligible bugs</h2>
         </dd>
       </dl>
 
-      <h2>Bug reporting</h2>
+      <h2 id="bug-reporting">Bug reporting</h2>
 
       <dl class="faq">
+        <dt id="what-next">Once I have found a vulnerability, what next?</dt>
+        <dd>
+          <p>The sooner we can reproduce and fix the bug, the sooner we can
+            fix it and send you your bounty. Please submit all bugs through
```

Comment:
" and send you your bounty" seems to imply all bugs get a bounty or least is something some might argue.  Maybe it's not all that important, but making note of the wording.

Reply:
it's fine as is

Reply:
This reads a little odd, as "fixing" is mentioned twice:

>The sooner we can reproduce and fix the bug, the sooner we can fix it...